services/web/serve-static.js

```diff
@@ -1,9 +1,14 @@
-const fs = require('fs');
-const url = require('url');
 const Koa = require('koa');
-const compress = require('koa-compress');
-const koaStatic = require('koa-static');
-const config = require('@kaareal/config');
+const koaLogger = require('koa-logger');
+const koaMount = require('koa-mount');
+const koaCompress = require('koa-compress');
+const koaBasicAuth = require('koa-basic-auth');
+const envMiddleware = require('./src/utils/middleware/env');
+const assetsMiddleware = require('./src/utils/middleware/assets');
```

Comment:
This is a bit wrong we should solve this with an nginx docker image, instead of running our own server. but yeah perhaps talking point for another day

Reply:
Yeah I remember your point from my other PR. I agree, but also we shouldn't require running an nginx server in our dev environment, so I think there's still a reason to have this.

Reply:
the reason why i build this out in the back in the days was that https://github.com/webpack/webpack-dev-server and other like it was like not maintained, but it looks like it has found new live.

Perhaps worth consider?
I specially like the idea about limiting the number of packages that we directly include.